src/extension/background-script/connectors/connector.example

```diff
@@ -34,7 +34,7 @@ class ConnectorExample implements Connector {
     // Add your own implementation.
   }
 
-  getBalance(): Promise<GetBalanceResponse> {
+  getBalance(): Promise<GetBalanceResponse> { // we could remove this
```

Comment:
We never use this. `balance` is being taken via `getInfo`

Reply:
I think we use the `getBalance()` function. 

also this is unrelated to this PR. let's check this separately, this PR is already big and adding a bunch of things.